scripts/tests/forest_tool_check.sh

```diff
@@ -5,6 +5,12 @@
 
 set -euxo pipefail
 
-FOREST_TOOL_PATH="forest-tool"
+source "$(dirname "$0")/harness.sh"
 
 "$FOREST_TOOL_PATH" state-migration actor-bundle
+
+# Exporting with an empty car should fail but not panic
+touch empty.car
+ERROR=$($FOREST_CLI_PATH archive export empty.car 2>&1 || true)
+assert_eq "$ERROR" "Error: input not recognized as any kind of CAR data (.car, .car.zst, .forest.car)"
+rm empty.car
```

Comment:
Can you write something like https://github.com/ChainSafe/forest/blob/62df2d8d608f2629664ad35e3e82fe3f930ee038/tests/import_snapshot_tests.rs?

```rust
touch("empty.car");
forest_cli().args(["archive export empty.car").assert().failure().stdout().contains("Error: input not recognized...")
```

Bonus points if you port the existing test and remove this file.

See also https://github.com/ChainSafe/forest/issues/2945